STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -651,15 +725,17 @@ We note that {{RFC8555}} does not define a registry for these objects.
 ## CSR Template Extensions
 {: #csr-template-registry }
 
-IANA is requested to establish a registry "STAR Delegation CSR Template Extensions", with "Expert Review" as its registration procedure.
+IANA is requested to establish a registry "STAR Delegation CSR Template
+Extensions", with "Expert Review" as its registration procedure.
 
 Each extension registered must specify:
 
 * An extension name
 * An extension syntax, as a JSON Schema snippet that defines a type
 * Mapping into an X.509 certificate extension.
 
-The initial contents of this registry are the extensions defined by the JSON Schema document in {{csr-template-schema}}.
+The initial contents of this registry are the extensions defined by the JSON
+Schema document in {{csr-template-schema}}.
```

Comment:
Please also update the document history section.

Reply:
will do.